app/models/refresh_state.rb

```diff
@@ -1,3 +1,5 @@
+require 'acts_as_tenant'
```

Comment:
@pkomanek should be rather included inside of a class like `ApplicationRecord` has